src/rpc/blockchain.cpp

```diff
@@ -2762,6 +2771,9 @@ UniValue CreateUTXOSnapshot(
     CCoinsStats stats{CoinStatsHashType::HASH_SERIALIZED};
     CBlockIndex* tip;
 
+    // used when human readable format is requested
+    const std::string& separator{","};
```

Comment:
Can do a `line_separator` (`"\n") here too...

```suggestion
    const auto separator = MakeByteSpan(std::string{","});
    const auto line_separator = MakeByteSpan(std::string{"\n"});
```

Reply:
Done.